src/main/java/io/kuzzle/sdk/core/Kuzzle.java

```diff
@@ -1823,13 +1827,39 @@ private void  cleanQueue() {
     }
   }
 
+  private void  mergeOfflineQueueWithLoader() {
+    KuzzleQueue<KuzzleQueryObject> additionalOfflineQueue = this.offlineQueueLoader.load();
```

Comment:
Since we leave the implementation of this method to SDK users, shouldn't this call be encapsulated in a `try..catch` statement, just in case?